src/main.py

```diff
@@ -22,19 +22,19 @@
 valid_path = '../mini_imagenet/csvsplits/valid.csv'
 test_path = '../mini_imagenet/csvsplits/test.csv'
 separator = ';'
-model_best_state_filename = 'best_state.pt'
+filename = 'best_protonet.pt'
 
 #*############################ 
 #*       Hyperparameters     #
 #*############################
-n_supports = 5
-n_queries = 5
-batch_size = 60
-learning_rate = 0.01
-momentum = 0.9
+n_support = 5
+n_query = 5
+n_samples_per_class = n_support + n_query
+learning_rate = 1e-3
 n_epochs = 600
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
-n_iterations = 100
+n_episodes = 100
+classes_per_it = (40, n_samples_per_class, n_samples_per_class)
```

Comment:
Le nombre de sample par classe c'est pas la même chose que le nombre de classe par itération. Ici, en validation et test, c'est un hasard que les deux valent 10.

Reply:
Donc en gros, je laisserais 10 hardcodé